src/Cli/dotnet/commands/dotnet-workload/update/WorkloadUpdateCommand.cs

```diff
@@ -65,9 +66,13 @@ internal class WorkloadUpdateCommand : CommandBase
             _includePreviews = parseResult.ValueForOption<bool>(WorkloadUpdateCommandParser.IncludePreviewsOption);
             _downloadToCacheOption = parseResult.ValueForOption<string>(WorkloadUpdateCommandParser.DownloadToCacheOption);
             _verbosity = parseResult.ValueForOption<VerbosityOptions>(WorkloadUpdateCommandParser.VerbosityOption);
-            _sdkVersion = string.IsNullOrEmpty(parseResult.ValueForOption<string>(WorkloadUpdateCommandParser.VersionOption)) ?
+            Option versionOption = new Option<string>("--sdk-version", Install.LocalizableStrings.VersionOptionDescription)
+            {
+                ArgumentHelpName = Install.LocalizableStrings.VersionOptionName
+            };
```

Comment:
Here too- why do we need to create a new option?

Reply:
same, reverting